lib/spack/spack/cmd/create.py

```diff
@@ -164,6 +168,10 @@ def install(self, spec, prefix):
         scons('prefix={0}'.format(prefix))
         scons('install')""",
 
+    'bazel': """\
+        # FIXME: Add logic to build and install here.
+        bazel('build')""",
```

Comment:
@adamjstewart One reason could be that `bazel` is designed to do part of the work that `spack` is doing : [download and install from external repositories](https://www.bazel.io/versions/master/docs/external.html) (and it seems at a first glance that this is part of its core business). In my opinion, we should at least read carefully what it does and see if we can integrate neatly before adding it to core files.

> The code added to build_environment.py is a single line, whereas it would take at least a few lines in the bazel package to do the same.

Yes, usually globals have that "great" property. Nonetheless the code added in a package scales (as it is distributed), the one in `build_environment` centralizes knowledge of each and every build system.